docs/DeveloperGuide.md

```diff
@@ -584,11 +584,20 @@ decided  to switch over to Alternative 1 for the user to easily view all the syn
 ### Appendix A: Product scope
```

Comment:
I suggest to put both the target user profile and value proposition directly under Appendix A.
This screenshot is taken from the AB3 DG.
![image](https://user-images.githubusercontent.com/69396486/140634411-37b0a878-5eac-4621-872d-e4a8b216f741.png)

Reply:
Yes, I agree. I have edited the DG accordingly.